src/System.Runtime/tests/System/String.cs

```diff
@@ -19,7 +20,7 @@ public static void TestLastIndexOfWithEmptyString()
     {
         string s = "Dill Guv Dill Guv Dill";
         int lastIndex = s.LastIndexOf("", StringComparison.OrdinalIgnoreCase);
-        Assert.True(lastIndex == s.Length - 1);
+        Assert.Equal(lastIndex, s.Length - 1);
```

Comment:
Thanks for changing all of these to be Equal instead of True.  However, the first argument to Equal is meant to be the expected value, the second the actual value, so most (all?) of these need to have their arguments swapped.  Otherwise, if the assert fails, the error message that gets output ends up being confusing.  Could you swap them where appropriate as well?